client/src/utils/parser.js

```diff
@@ -1,8 +1,43 @@
+/**
+ * @description Parses an JSON created from the .bpmn xml of the model to an executable .robot file
+ * @returns {string} Code that has to be put in .robot file
+ */
+const parseDiagramJson = (json_data) => {
```

Comment:
Was not only wondering in this file, but did we ever decide against functions? What are the benefits of only using arrow functions?

Reply:
Thats the way that it was done in the react tutorial I watched. So I just thought to be consistent it would be a good idea to use arrow functions everywhere.

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions
I am used to arrow functions not being used for methods

Reply:
For now I would leave it like that because in parser.js we only have functions and arrow functions shouldn't be a problem. I also just read up on it a little bit and what I found so far suggests that arrow functions should be used with react components as well.

Reply:
IMO that is not something that either you or me should decide. Because this is a code style issue, I would want to have this discussed with the whole team to keep consistency for the future.